### PR TITLE
add doc about webhook

### PR DESCRIPTION
**Pull request by zhaojkun**
@wooparadog

### CODE REVIEW

File: docs/web-api.md [hunk 1]
```diff
@@ -501,28 +501,64 @@ Event
...
+  PayLoad :
+  {
+    "id": "ab372c88c08ada73bd35feea740896d340de770e",
+    "index": {
+      "name": "timer.count_ps.note.add",
+      "stamp": 1477361415,
+      "score": 0,
+      "average": 1111,
+      "link": 1,
+      "matchedRules": null
```
Comment (wooparadog) [inline docs/web-api.md]: could be deleted?

File: docs/web-api.md [hunk 2]
```diff
@@ -501,28 +501,64 @@ Event
...
+    },
+    "project": {
+      "id": 14,
+      "name": "note",
+      "enableSilent": true,
+      "silentTimeStart": 0,
+      "silentTimeEnd": 6,
+      "teamID": 4
+    },
+    "user": null,
```
Comment (wooparadog) [inline docs/web-api.md]: could be deleted?

File: docs/web-api.md [hunk 3]
```diff
@@ -501,28 +501,64 @@ Event
...
+    "ruleTranslatedComment": "add note"
```
Comment (wooparadog) [inline docs/web-api.md]: is it processed or raw format? this needs to be formatted message.

Comment (zhaojkun) [inline docs/web-api.md]: this message has been processed

**Comment by wooparadog** [comment]
@tevino

**Pull request merged**